.jshintrc

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Why would we like to use two different types of linter?

Reply:
`jshint` is utilized in huge amounts legacy codes.

The .jshintignore, .jshintrc files are auto generated.

I'll remove them in this change.